packages/labextension/src/plugin.ts

```diff
@@ -1,69 +1,46 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-
 import type {
-  JupyterFrontEndPlugin, JupyterFrontEnd
+  JupyterFrontEndPlugin,
```

Comment:
adding https://github.com/trivago/prettier-plugin-sort-imports , with a bit of config, can help _substantially_ for big import lists e.g. "first unknown-third-party, then lumino, then lab, then local"